Lib/test/test_traceback.py

```diff
@@ -3132,13 +3133,13 @@ class TestFrame(unittest.TestCase):
 
     def test_basics(self):
         linecache.clearcache()
-        linecache.lazycache("f", globals())
-        f = traceback.FrameSummary("f", 1, "dummy")
+        linecache.lazycache(test_path, globals())
```

Comment:
We don't want `linecache` to possibly pollute the cache of this file during the test.